projects/packages/my-jetpack/_inc/components/upsell-banner/index.jsx

```diff
@@ -0,0 +1,82 @@
+import { Gridicon, Button } from '@automattic/jetpack-components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+import { CardWrapper } from '../card';
+
+import './style.scss';
+
+/**
+ * Upsell banner component.
+ *
+ * @param {object} props - Component props.
+ * @returns {object} UpsellBanner React component.
+ */
+const UpsellBanner = props => {
```

Comment:
Ideally, this component should be extracted and moved to `@automattic/jetpack-components` (`js-package/components/components`). It could be used by other existing banners in the Jetpack plugin. It will need the Card component (also it'll need to be extracted from Jetpack plugin into jetpack-components) instead of using the CardWrapper component.